sw-emulator/lib/periph/src/hash_sha256.rs

```diff
@@ -99,6 +103,18 @@ pub struct HashSha256 {
 
     op_complete_action: Option<ActionHandle>,
 }
+#[derive(Debug)]
+pub struct WntzParams {
+    wntz_mode: bool,
```

Comment:
Suggest adding descriptions to the Winternitz parameters.

Reply:
Done